README.md

```diff
@@ -174,54 +158,116 @@ The `ScheduleEntry` object contains the following name/value pairs:
+|`units`|true|string|N/A|Data units, e.g., `"dBm"`, `"watts"`, `"volts"`.|
+|`reference`|false|string|N/A|Data reference point, e.g., `"receiver input"`, `"antenna output"`, `"output of isotropic antenna"`.|
+
+#### FrequencyDomainDetection Object
+Frequency-domain detection algorithms are applied to discrete Fourier transforms of IQ time series captured at a single frequency. The `FrequencyDomainDetection` object contains the following name/value pairs:  
+
+|name|required|type|unit|description|
+|----|--------------|-------|-------|-----------|
+|`detector`|true|string|N/A|E.g. `"fft_sample_iq"`, `"fft_sample_power"`, `"fft_mean_power"`, `"fft_max_power"`, `"fft_min_power"`, `"fft_median_power"`, `"fft_m4s_power"`.|
 |`number_of_ffts`|true|integer|N/A|Number of FFTs to be integrated over by detector.|
 |`units`|true|string|N/A|Data units, e.g., `"dBm"`, `"watts"`, `"volts"`.|
 |`reference`|false|string|N/A|Data reference point, e.g., `"receiver input"`, `"antenna output"`, `"output of isotropic antenna"`.|
+|`number_of_samples_in_fft`|true|integer|N/A|Number of samples in FFT to calcluate delta_f = [`samplerate`](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#global-object)/`number_of_samples_in_fft`.|
+|`window`|true|string|N/A|E.g. `"blackman-harris"`, `"flattop"`, `"gaussian_a3.5"`, `"gauss top"`, `"hamming"`, `"hanning"`, `"rectangular"`.|
+|`equivalent_noise_bandwidth`|false|float|Hz|Bandwidth of brickwall filter that has same integrated noise power as that of the actual filter.|
 
-##### SteppedFrequencyFFTDetection Object
-The `SteppedFrequencyFFTDetection` object contains the following name/value pairs:  
+#### SteppedFrequencyMeasurement Object
+The `SteppedFrequencyMeasurement` object contains the following name/value pairs:  
 
 |name|required|type|unit|description|
 |----|--------------|-------|-------|-----------|
-|`center_frequency_start`|true|float|Hz|First center frequency of scan.|
-|`center_frequency_stop`|true|float|Hz|Last center frequency of scan.|
-|`center_frequency_step`|true|float|Hz|Center frequency step of scan.|
-|`single_frequency_fft_detection`|true|object|N/A|See [SingleFrequencyFFTDetection Object](#singlefrequencyfftdetection-object) definition.|
+|`frequencies`|false|array of floats|Hz|Center frequencies where stepped-frequency detections are performed.|
+|`algorithm`|true|object|N/A|Algorithm applied to IQ samples. See [TimeDomainDetection Object](#timedomaindetection-object), [FrequencyDomainDetection Object](#frequencydomaindetection-object) definition.|
+
+Example `SteppedFrequencyMeasurement` object for case where mean-power measurements are performed at five frequencies:
+
+```
+{
+  "frequencies": [100000000, 200000000, 300000000, 400000000, 500000000]
+  "algorithm"
+  {
+    "detector": "mean_power",
+    "number_of_samples": 100000,
+    "units": "dBm",
+    "reference": "output of isotropic antenna"
+  }
+}
+```
```

Comment:
1) This isn't valid JSON (missing comma after frequencies array, missing colon after algorithm). I recommend dropping any JSON examples into an online JSON validator/formatter just to make sure I didn't miss anything as well. 

2) I think this is trying to be just a bit too concise to the point of making it difficult to interpret. Remember that when you use a `TimeDomainDetection` object or a `FrequencyDomainDetection` object, they're not actually labeled as such. So I'm looking at this and thinking "okay so to figure out if this is a time domain or frequency domain stepped measurement, I'm going to need to look at the different fields in both objects, pick a field that's required in one but doesn't exist in the other and look for that"... etc.

Quick off-the-cuff suggestion: `detection_domain: "time"|"frequency"`?